config/integration.exs

```diff
@@ -2,8 +2,10 @@ import Config
 
 config :elsa, brokers: [localhost: 9092]
 
-config :elsa,
-  divo: [
-    {DivoKafka, [create_topics: "elsa-topic:2:1", outside_host: "localhost"]}
-  ],
+config :elsa_fi,
```

Comment:
I think the other configs also need this changed as well?

Reply:
Yeah, so I can make everything :elsa_fi, which means I need to change all the Application.get_env(:elsa, blah blah) stuff in the code.  Which means it gets awkward if we ever want to merge back with the main fork.  That being said, we are getting warnings for this so fixing it is probably the lesser evil.